analyses/optimus/verify_biological_validity_of_optimus_outputs.ipynb

```diff
@@ -0,0 +1,250 @@
+{
```

Comment:
apparently, there is a peak shifting between 10X to Optimus. it looks like Optimus may show lower coverage or lower UMI count compare to 10X cellrange. What if you do
1) just do xyplot to show the library size difference. 
or 
2) test out distribution by using KS or something like it. 
or
3) do a quick clustering and check the similarity in clustering results, you can calculate something adjusted Rand index 
I think the approach is right and unfortunately(or fortunately), we do see difference or discrepancy between pipelines. I am wondering what cause it. 

Anyway, it is really exciting to see results produced by Optimus.